src/components/nodes/signalStartEvent/index.js

```diff
@@ -24,4 +25,23 @@ export default merge(cloneDeep(startEventConfig), {
   validateIncoming() {
     return false;
   },
+  inspectorConfig: [
+    {
+      items: [
+        {
+          items: [
+            {},
+            {
+              component: SignalSelect,
+              config: {
+                label: 'Signal',
+                name: 'signalRef',
+                helper: 'Signal that will trigger this start event',
+              },
+            },
```

Comment:
Minor nice-to-have nitpick: this shows up a few times - you can maybe extract this and and have the correct helper text passed into your extracted object; just to keep it DRY

Reply:
https://github.com/ProcessMaker/modeler/pull/1176#discussion_r408157617
@johnblackspear what do you mean?
Well `window.ProcessMaker.$modeler` is a reference to the `modeler` vue component that is available in ProcessMaker project and is used to get the signal nodes definitions from SignalSelect, since this component can not receive the definitions as parameter.

```
window.ProcessMaker.$modeler = this.$refs.modeler;
```

Reply:
The snippet that is used to configure the inspector to show a signal select occurs in a few places (e.g. `boundarySignalEvent`, `signalEndEvent`, here). So my suggestion is to extract it out so that you don't have to repeat yourself.

We already do something similar for `advancedAccordionConfig` - you can see how `idConfigSettings` is used in it. The same can be done for signal select, for example (you'll need to edit this to your taste, I'm typing train-of-thought in this comment box) :

```javascript
import SignalSelect from '@/components/inspectors/SignalSelect';

const signalSelectConfig = {
  component: SignalSelect,
  config: {
    label: 'Signal',
    name: 'signalRef',
    helper: 'Please choose the Signal that will trigger this',
  },
};

export default signalSelectConfig;
```

Now you can re-use this config in the relevant places through an import and make changes in a single location. Your `inspectorConfig` would in this example look something like:
```javascript
import signalSelectConfig from 'locationYouChoseForThis';

//stuff...
inspectorConfig: [
    {
      items: [
        {
          items: [
            {},
            signalSelectConfig,
          ],
        },
      ],
    },
  ],
//etc..
```

BTW, I don't think that you really need different helper text for each case that you use the signalSelect if you write a generic helper, but if you do then it would be simple to pass it in through a minor change to the example above:

```javascript
const signalSelectConfig = (helper) => {
  return {
    component: SignalSelect,
    config: {
      label: 'Signal',
      name: 'signalRef',
      helper,
    },
  };
};
```

Reply:
> [#1176 (comment)](https://github.com/ProcessMaker/modeler/pull/1176#discussion_r408157617)
> @johnblackspear what do you mean?
> Well `window.ProcessMaker.$modeler` is a reference to the `modeler` vue component that is available in ProcessMaker project and is used to get the signal nodes definitions from SignalSelect, since this component can not receive the definitions as parameter.
> 
> ```
> window.ProcessMaker.$modeler = this.$refs.modeler;
> ```

Hi @caleeli just realised that you reply links to another comment here, missed that, sorry :)

My question about the status is because @chipit24 asked that you please access `rootElements` via the Vuex store - wondered how that was coming along.